codecov.yml

```diff
@@ -5,3 +5,5 @@ coverage:
         target: auto
         threshold: 5%
         only_pulls: true
+    patch:
+      threshold: 5%
```

Comment:
Should this line not go under a `default` key? Also, does this mean it will allow PRs with pull request having patch coverage upto 95% ?

Reply:
yes, this should be inside the default. `threshold` mean x % drop is allowed from the main. I think the target is used for the coverage upto 95  https://docs.codecov.com/docs/commit-status#threshold https://docs.codecov.com/docs/commit-status#target
```
patch:
    default:
        threshold: 5%
```
You can use `curl -X POST --data-binary @codecov.yml https://codecov.io/validate` to validate codecov.yml

Reply:
but I'm not sure we will merge this or not, most probably I'll close the PR